cheerio-scraper/INPUT_SCHEMA.json

```diff
@@ -197,6 +197,24 @@
 			"default": {},
 			"prefill": {},
 			"editor": "json"
+		},
+		"datasetName": {
+			"title": "Dataset Name",
```

Comment:
Only capitalize the first letter. => `Dataset name`. Same in the other storages.